src/main/java/com/blackduck/integration/stepworkflow/jenkins/JenkinsStepWorkflow.java

```diff
@@ -82,5 +102,16 @@ protected PhoneHomeRequestBody addJenkinsMetadataAndBuildPhoneHomeRequest(PhoneH
         return phoneHomeRequestBodyBuilder.build();
     }
 
+    protected PhoneHomeCredentials getGa4Credentials() throws IOException, InterruptedException, JsonSyntaxException, IntegrationException {
+        String fileUrl = CREDENTIALS_PATH;
+        logger.debug("Downloading phone home credentials.");
+        RequestBuilder createRequestBuilder = intHttpClient.createRequestBuilder(HttpMethod.GET);
+        HttpUriRequest request = createRequestBuilder
+                .setUri(fileUrl)
+                .build();
+        Response response = intHttpClient.execute(request);
+        return gson.fromJson(response.getContentString(), PhoneHomeCredentials.class);
+    }
+
```

Comment:
I checked that we do have a concept of using "SNAPSHOT" in the versioning of this project. It would be best to implement the method as is defined in the Detect source code which sets a `TEST_CREDENTIALS_PATH` for a test environment (i.e. if version contains SNAPSHOT or SIGQA). Even though the tests are mocking the phone home operation, it would be safer to add test credentials to avoid any future chances of the prod endpoint receiving any test data.

Reply:
Okay, I will add that method.